### PR TITLE
dev-libs/libzip: version bump 1.5.1

### PR DESCRIPTION
**Pull request by hungte**
Upgrade libzip from 1.3.0 to 1.5.1.

### CODE REVIEW

**Comment by hungte** [comment]
@vapier this is the PR.

**Comment by hungte** [comment]
e3f230e  includes signing

File: dev-libs/libzip/libzip-1.5.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+HOMEPAGE="https://nih.at/libzip/"
+SRC_URI="https://www.nih.at/libzip/${P}.tar.xz"
+
+LICENSE="BSD"
+SLOT="0/5"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~sparc ~x86 ~amd64-fbsd ~x86-fbsd ~amd64-linux ~x86-linux ~x86-macos"
+IUSE="bzip2 static-libs"
+
+RDEPEND="
+	sys-libs/zlib
```
Comment (hydrapolic) [inline dev-libs/libzip/libzip-1.5.1.ebuild]: Perhaps a slot operator to rebuild on zlib changes?

File: dev-libs/libzip/libzip-1.5.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (Zlogene) [inline dev-libs/libzip/libzip-1.5.1.ebuild]: why not 7?

File: dev-libs/libzip/libzip-1.5.1.ebuild (hunk 1 above)
Comment (a17r) [reply dev-libs/libzip/libzip-1.5.1.ebuild]: Please don't, the zlib subslot was a mistake in the first place.

File: dev-libs/libzip/libzip-1.5.1.ebuild (hunk 2 above)
Comment (hungte) [reply dev-libs/libzip/libzip-1.5.1.ebuild]: Updated to 7.

**Comment by hungte** [comment]
> There seem to be some unhandled dependencies here, as seen in CMakeLists.txt:
> OPTION(ENABLE_GNUTLS "Enable use of GnuTLS" ON)
> OPTION(ENABLE_OPENSSL "Enable use of OpenSSL" ON)

  Added to dependency.

> OPTION(ENABLE_COMMONCRYPTO "Enable use of CommonCrypto" ON)

  Common crypto is currently not available in portage, but the system will still use it if found.
  i.e., the ON here actually means "auto-detect" so there won't be dependency issue on portage.

**Comment by a17r** [comment]
Thanks for your work, unfortunately the port to cmake requires bigger changes not feasible to deal with in a PR. I've got it mostly ready and will push it to tree after some testing.

**Pull request closed without merging**